platform/ios/display_server_ios.mm

```diff
@@ -442,6 +442,24 @@
 	}
 }
 
+void DisplayServerIOS::ios_set_ui_state(bool p_hide_home_indicator, bool p_suppress_bottom_ui_gesture, bool p_suppress_top_ui_gesture, bool p_hide_status_bar, float p_status_bar_fade_time) {
+	ProjectSettings::get_singleton()->set_setting("display/window/ios/hide_home_indicator", hide_home_indicator);
```

Comment:
No platform is setting project settings, so it probably should read project setting values to the DisplayServerIOS variables on startup and only update them.